Entity/OverrideLeadFieldRepository.php

```diff
@@ -32,10 +40,12 @@ class OverrideLeadFieldRepository extends LeadFieldRepository
      * @param ClassMetadata      $class
      * @param ExtendedFieldModel $fieldModel
      */
-    public function __construct(EntityManager $em, ClassMetadata $class, ExtendedFieldModel $fieldModel)
+    public function __construct(EntityManager $em, ClassMetadata $class, ExtendedFieldModel $fieldModel, CoreParametersHelper $coreParametersHelper)
     {
         parent::__construct($em, $class);
-        $this->fieldModel = $fieldModel;
+        $this->fieldModel                  = $fieldModel;
+        $this->coreParametersHelper        = $coreParametersHelper;
+        $this->extendedFieldConfigurations = $this->getExtendedFieldConfigurations();
```

Comment:
I don't think we should ever put something that requires a query in a constructor. Rather have it populate as needed (and cache). Not all mautic actions require this array.